insights/parsers/qpid_stat.py

```diff
@@ -71,39 +106,21 @@ class QpidStatQ(CommandParser):
         []
     """
 
-    def __iter__(self):
-        return iter(self.data)
-
     def parse_content(self, content):
+        content = super(QpidStatQ, self).parse_content(content)
+
         self.data = parse_fixed_table(
-            [line for line in content if '========' not in line],
+            [line for line in content],
             heading_ignore=['queue']
         )
         self.by_queue = dict(
             (q['queue'], q)
             for q in self.data
         )
```

Comment:
For this class and `QpidStatU` I think you could just pull all of `parse_content` into the base class and just use `pass` for these two classes.  Then just completely overload `parse_content` in `QpidStatG`.

Reply:
@bfahr - we have made changes as per your suggestions, please check it again. Thanks.